x-pack/legacy/plugins/uptime/server/lib/helper/get_histogram_interval.ts

```diff
@@ -20,7 +25,11 @@ export const getHistogramInterval = (
   if (to === undefined) {
     throw Error('Invalid dateRangeEnd value');
   }
-  return `${Math.round(
+  const interval = Math.round(
     (to.valueOf() - from.valueOf()) / (bucketCount || QUERY.DEFAULT_BUCKET_COUNT)
-  )}ms`;
+  );
+  return {
+    interval,
+    intervalFormatted: `${interval}ms`,
```

Comment:
This is a weird pattern, returning two versions of it. Can we just make second function `getHistogramIntervalFormatted` that calls `getHistogramInterval` and appends `"ms"`?

Reply:
++ I am fine with that

Reply:
Resolved by https://github.com/elastic/kibana/pull/48157/commits/de6df2db5969dd4fd730e93e2795d04942fdb642.